docs/introduction.md

```diff
@@ -97,7 +97,7 @@ You can download Ant Media Server Community Edition from [Github](https://githu
 Ant Media Server has two types of licenses.
 
 1. Ant Media Server Community Edition is free to use, with Apache license.
-2. Ant Media Server Enterprise Edition needs a paid license per instance/server. Paid license options are hourly, monthly, annually and perpetual and can purchase a license directly from [antmedia.io](https://antmedia.io). 
+2. Ant Media Server Enterprise Edition needs a paid license per instance/server. Paid license options are monthly, annually and perpetual and you can purchase a license directly from [antmedia.io](https://antmedia.io). 
```

Comment:
Just a suggestion, I thought it could read a bit better:  Ant Media Server Enterprise Edition requires a license, which can be purchased through antmedia.io as a monthly, annual, or perpetual subscription.